plugins/wporg-learn/wporg-learn.php

```diff
@@ -33,6 +34,59 @@
 add_action( 'init', array( 'WPORG_Learn\Lesson_Plan', 'lesson_instruction_type_taxonomy' ) );
 add_filter( 'the_content', array('WPORG_Learn\Lesson_Plan', 'replace_image_links' ) );
 
+
+add_action( 'init', array( 'WPORG_Learn\Workshop', 'workshop_post_type' ) );
+add_action( 'init', array( 'WPORG_Learn\Workshop', 'lesson_workshop_taxonomy' ) );
+add_action( 'init', array( 'WPORG_Learn\Workshop', 'workshop_topics_taxonomy' ) );
+add_filter('query_vars', 'add_category');
+add_action('init', 'string_url_rewrite', 10, 0);
+add_filter( 'excerpt_length', 'theme_slug_excerpt_length', 999 );
+
+/**
+ * Add a query parameter for use with the lesson-plan/workshop search directory
+ * @param array $vars
+ * @return array
+ */
+function add_category( $vars ) {
+	$vars[] = 'category';
+	return $vars;
+}
+
+/**
+ * Creates rewrites that are used in the lesson plan/workshop directory.
+ */
```

Comment:
You may be right here. 

I was led to rewrites because paths like `workshops/featured` and `lesson-plans/latest` are great for SEO and SEO is critical for this theme. Since `workshops` and `lesson-plans` are the archive names on the custom post type, this was the only way I could figure out how to do it.

I also considered using Categories and changing the category base. However, because we are dealing with the two custom post types in the same manner, I couldn't set `(workshops|lesson-plan)` for the category base. That is not supported.